guideforms.py

```diff
@@ -117,7 +117,11 @@
          'our <a target="_blank" href="'
          'https://docs.google.com/document/d/'
          '1RC-pBBvsazYfCNNUSkPqAVpSpNJ96U8trhNkfV0v9fk/edit">'
-         'principles of web compatibility</a>.')),
+         'principles of web compatibility</a>.<br>'
+         '<br>'
+         'Please include citation links below where possible. Examples include '
+         'resolutions from relevant standards bodies (e.g. W3C Working Group), '
```

Comment:
```suggestion
         'resolutions from relevant standards bodies (e.g. W3C working group), '
```

Reply:
'Working group' is not a proper noun if you're not referencing the name of a specific working group.

Reply:
Done.